### DISCUSSION TITLE
Great work

### DISCUSSION THREAD

**Discussion by Runner3131** [muhammadanas0716/Machine-Learning-101#68] [General]
Fantastic job so far. I will be able to get some cheat sheets posted up in the next few days. I'm also in the process of starting a blog/website about web development and promoting my web development company DevelopXcloud. and was wondering if it was okay if I linked to your repo in one of my posts, you have a interesting repo here.
Thanks
DevelopXcloud.

**Comment by muhammadanas0716** [comment]
Absolutely no problem. If it's helping people break into tech, no harm in adding it.
